internal/pkg/agent/application/upgrade/step_download.go

```diff
@@ -40,6 +40,10 @@ func (u *Upgrader) downloadArtifact(ctx context.Context, version, sourceURI stri
 		}
 	}
 
+	u.log.Infow("Downloading upgrade artifact", "version", version,
```

Comment:
I thought that the downloader emitted periodic lines, is this needed here as well?

Reply:
Yeah this one can be removed.